src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/StartToolStep.groovy

```diff
@@ -207,7 +207,9 @@ class StartToolStep extends Step {
                     if (ProcessUtil.killTTProcesses(timeout)) {
                         listener.logger.println("Stopped tracetronic tools successfully.")
                     } else {
-                        throw new TimeoutException("Timeout of ${this.timeout} seconds exceeded for stopping tracetronic tools!")
+                        throw new AbortException(
+                                "Timeout of ${this.timeout} seconds exceeded for stopping tracetronic tools!" +
+                                "Please ensure that tracetronic tools are not already stopped.")
```

Comment:
It is also possible that some process is blocked and we run into a timeout as a result